core/src/bundle/usBundleUtils.cpp

```diff
@@ -183,6 +310,11 @@ void* GetSymbol(const BundleInfo& bundle, const char* symbol)
 {
   return GetSymbol_impl(bundle, symbol);
 }
+  
+bool IsCurrentExecutable(const std::string& filepath)
+{
+  return execInfo.path == filepath || execInfo == FileInfo(filepath); // Avoid constructing the FileInfo object if paths match
```

Comment:
A comment about why `|| execInfo == FileInfo(filepath);` is here would be useful.